rosdep/base.yaml

```diff
@@ -1028,6 +1028,14 @@ flex:
   opensuse: [flex]
   rhel: [flex]
   ubuntu: [flex]
+flite:
+  alpine: [flite]
+  arch: [flite]
+  debian: [flite]
+  fedora: [flite]
+  gentoo: [app-accessibility/flite]
+  rhel: [alpine]
```

Comment:
this line doesn't look correct

Reply:
oh, I guess it's only a coincidence lol https://src.fedoraproject.org/rpms/alpine

Reply:
Sorry it's my mistake.
Thank you!